### ISSUE TITLE
N v1.3c (pre-AS3 rewrite) refuses to let player start levels

### ISSUE THREAD

**Issue by WumboSpasm** [ruffle-rs/ruffle#1645]
File: [n.zip](https://github.com/ruffle-rs/ruffle/files/5396920/n.zip)

When issue #1343 was published, the game didn't load buttons on the title screen, making it unplayable. That issue has since been fixed, but now it won't recognize spacebar presses needed to start levels. Oddly, pressing Q to return to the level selection works fine.

The graphical bugs mentioned in the original issue still apply (player body is mostly but sometimes completely invisible, and blood doesn't appear when you die).

(tested in the Ruffle desktop app)

**Comment by Toad06** [comment]
So if I'm correct, the issue is that the condition `if(APP_KEY_TRIG && Key.isDown(32))` in `function App_Tick_PreLevelPause()` is never met because `APP_KEY_TRIG` is always false.

There's only one place where `APP_KEY_TRIG` can be set to true:
```
_root.onKeyDown = function()
{
   if(!APP_KEY_PRESSED)
   {
      APP_KEY_TRIG = true;
   }
   else
   {
      APP_KEY_TRIG = false;
   }
   APP_KEY_PRESSED = true;
};
```

`_root.onKeyDown` is correctly executed... but for some reason, it's executed twice! So when you press space, `APP_KEY_TRIG` is set to true but immediately sets back to false.

(However, even if you workaround this, the game has collision issues.)

**Comment by Herschel** [comment]
Fixed by #2122
